insights/parsers/ovs_vsctl_show.py

```diff
@@ -1,90 +1,69 @@
-                options: {peer=int-br-ctlplane}
-    Bridge br-int
-        fail_mode: secure
-        Port br-int
-            Interface br-int
-                type: internal
-        Port int-br-ctlplane
-            Interface int-br-ctlplane
-            type: patch
-            options: {peer=phy-br-ctlplane}
-        Port "tap293d29ec-d1"
-            tag: 1
-            Interface "tap293d29ec-d1"
-                type: internal
+                options: {peer=phy-br-ctlplane}
+            Port "tap293d29ec-d1"
+                tag: 1
+                Interface "tap293d29ec-d1"
+                    type: internal
+        Bridge br-tun
+            fail_mode: secure
+            Port "vxlan-aca80118"
+                Interface "vxlan-aca80118"
+                    type: vxlan
+                    options: {df_default="true", in_key=flow, local_ip="172.168.1.26", out_key=flow, remote_ip="172.168.1.24"}
+            Port "vxlan-aca80119"
+                Interface "vxlan-aca80119"
+                    type: vxlan
+                    options: {df_default="true", in_key=flow, local_ip="172.168.1.26", out_key=flow, remote_ip="172.168.1.25"}
     ovs_version: "2.3.2"
 
-   Parse result:
-   ===============================
-    {
-        "uuid": "aa3f16d3-9534-4b74-8e9e-8b0ce94038c5"
-        "bridges": {
-            "br-ctlplane": {
-                "ports": [
-                    {
-                        "interface": "eth0",
-                        "port": "eth0"
-                    },
-                    {
-                        "interface": "br-ctlplane",
-                        "port": "br-ctlplane",
-                        "type": "internal"
-                    },
-                    {
-                        "interface": "phy-br-ctlplane",
-                        "options": {
-                            "peer": "int-br-ctlplane"
-                        },
-                        "port": "phy-br-ctlplane",
-                        "type": "patch"
-                    }
-                ]
-            },
-            "br-int": {
-                "fail_mode": "secure",
-                "ports": [
-                    {
-                        "interface": "br-int",
-                        "port": "br-int",
-                        "type": "internal"
-                    },
-                    {
-                        "interface": "int-br-ctlplane",
-                        "options": {
-                            "peer": "phy-br-ctlplane"
-                        },
-                        "port": "int-br-ctlplane",
-                        "type": "patch"
-                    },
-                    {
-                        "interface": "tap293d29ec-d1",
-                        "port": "tap293d29ec-d1",
-                        "tag": "1",
-                        "type": "internal"
-                    }
-                ]
-            }
-        },
-        "ovs_version": "2.3.2",
-    }
```

Comment:
@PaulWay I realize this is verbose, but I feel like removing it makes it harder to understand the format of the output.  It takes some imagination to see this in the examples.  If you remove this could you show more detail in the examples sections to demonstrate the structure of the output?

Reply:
A fair point.  It's always the problem with this kind of structured information - it's hard to see what you're getting :-)  I'm not convinced that the output structure is that unintuitively different from the original indented input that it's hard to understand what's available, but I will add more examples :-)

Reply:
Let me know what you think of that now, @bfahr